src/enzo/ReadParameterFile.C

```diff
@@ -78,29 +80,38 @@ int ReadParameterFile(FILE *fptr, TopGridData &MetaData, float *Initialdt)
   char **active_particle_types;
   active_particle_types = new char*[MAX_ACTIVE_PARTICLE_TYPES];
   int active_particles = 0;
-  
+
+#ifdef USE_GRACKLE
```

Comment:
I think this `ifdef` needs to be removed because there is the code inside `GrackleSetDefaultParameters` that checks for `use_grackle = 1` and grackle not compiled.

Reply:
Agreed -- good catch!